dango/httpd/src/graphql.rs

```diff
@@ -43,6 +43,15 @@ pub fn build_schema(app_ctx: Context) -> AppSchema {
         tokio::spawn,
     );
 
+    let file_transaction_loader: DataLoader<
+        dataloader::file_transaction::FileTranscationDataLoader,
+    > = DataLoader::new(
+        dataloader::file_transaction::FileTranscationDataLoader {
+            indexer: app_ctx.indexer_path.clone(),
+        },
+        tokio::spawn,
+    );
+
```

Comment:
Since you use `indexer/httpd` graphql, you should also add the dataloader in `indexer/httpd/src/graphql.rs` as well.